enforcer/README.md

```diff
@@ -147,17 +169,21 @@ behalf of the Enforcer.
         //   string - a human readable error returned by the plugin. valid
         //     if false is returned.
         Remove(id int) (bool, string)
+
+COMMENT: There must be an Answer() API here somewhere?  Is it the same as the one above?
```

Comment:
No, because we're delegating policy enforcement (Application) to the plugin.

Reply:
So Padme includes its own policy engine with an API for asking for policy decisions that only works with its policy engine.  I thought for sure one of the goals of Padme was to include a standard interface for asking for policy decisions from many different policy engines.  So for example, instead of the client knowing different APIs for asking for decisions from OPA and XACML, the client would know the Padme API, and Padme would ask OPA/XACML under the hood.  But that seems like it's out of scope for Padme. Am I wrong?